src/selectors/entities/common.js

```diff
@@ -2,8 +2,9 @@
 // See License.txt for license information.
 // @flow
 
-import type {GlobalState} from '../../types/store';
-import type {UserProfile} from '../../types/users';
+import type {GlobalState} from 'mattermost-redux/types/store';
```

Comment:
So you are saying that `src/types/store` won’t work here?

This is sending mix signals to other developers, that is why I want us to be consistent

Reply:
Yeah, the problem is we have the compiled version, which simply compile to mattermost-redux/whatever, and the flow-types, which can be follow the same pattern from the outside, but can't follow the same pattern in the inside (with the current approach). We can rethink it, flow integration is a Work in Progress, so we have to polish it.